classes/01class/exercises/c01-aws01/gepifanio/README.md

```diff
@@ -0,0 +1,362 @@
+            "Architecture": "x86_64",
+            "BlockDeviceMappings": [],
+            "ClientToken": "",
+            "EbsOptimized": false,
+            "Hypervisor": "xen",
+            "NetworkInterfaces": [
+                {
+                    "Attachment": {
+                        "AttachTime": "2020-06-13T04:10:11+00:00",
+                        "AttachmentId": "eni-attach-01de62fa0eaebedde",
+                        "DeleteOnTermination": true,
+                        "DeviceIndex": 0,
+                        "Status": "attaching"
+                    },
+                    "Description": "",
+                    "Groups": [
+                        {
+                            "GroupName": "my-sg",
+                            "GroupId": "sg-03e80096433013264"
+                        }
+                    ],
+                    "Ipv6Addresses": [],
+                    "MacAddress": "02:84:38:7d:03:00",
+                    "NetworkInterfaceId": "eni-067f8b30048f4bc63",
+                    "OwnerId": "873547843801",
+                    "PrivateDnsName": "ip-172-31-16-12.us-west-2.compute.internal",
+                    "PrivateIpAddress": "172.31.16.12",
+                    "PrivateIpAddresses": [
+                        {
+                            "Primary": true,
+                            "PrivateDnsName": "ip-172-31-16-12.us-west-2.compute.internal",
+                            "PrivateIpAddress": "172.31.16.12"
+                        }
+                    ],
+                    "SourceDestCheck": true,
+                    "Status": "in-use",
+                    "SubnetId": "subnet-2df43655",
+                    "VpcId": "vpc-fc9fee84",
+                    "InterfaceType": "interface"
+                }
+            ],
+            "RootDeviceName": "/dev/sda1",
+            "RootDeviceType": "ebs",
+            "SecurityGroups": [
+                {
+                    "GroupName": "my-sg",
+                    "GroupId": "sg-03e80096433013264"
+                }
+            ],
+            "SourceDestCheck": true,
+            "StateReason": {
+                "Code": "pending",
+                "Message": "pending"
+            },
+            "VirtualizationType": "hvm",
+            "CpuOptions": {
+                "CoreCount": 1,
+                "ThreadsPerCore": 1
+            },
+            "CapacityReservationSpecification": {
+                "CapacityReservationPreference": "open"
+            },
+            "MetadataOptions": {
+                "State": "pending",
+                "HttpTokens": "optional",
+                "HttpPutResponseHopLimit": 1,
+                "HttpEndpoint": "enabled"
+            }
+        }
+    ],
+    "OwnerId": "873547843801",
+    "ReservationId": "r-01624fca0dd7b9813"
+}
+
+
+```
+
+- Commands to connect to the second EC2 instance:
+
+```
+ssh -i "MyKeyPair.pem" ubuntu@ec2-52-12-177-52.us-west-2.compute.amazonaws.com
```

Comment:
this instance should not have a public IP address. can you review this part?

Reply:
Fixed, thank you!